### DISCUSSION TITLE
Use blue instead of red for LinkedIn's notification badges

### DISCUSSION THREAD

**Discussion by brandon-mork** [jordwest/news-feed-eradicator#445] [Ideas]
>Red is a trigger color that instantly draws our attention.

from [Center for Humane Technology](https://www.humanetech.com/take-control) (this extension is referenced further down on the linked page! 😄)

**Comment by aaditkamat** [comment]
It's quite interesting that I'd come across this exact Facebook feature change through the book [Digital Minimalism by Cal Newport](https://books.google.com.sg/books?id=uS9eDwAAQBAJ&printsec=frontcover&dq=digital+minimalism+choosing+a+focused+life+in+a+noisy+world&hl=en&sa=X&redir_esc=y#v=onepage&q=blue%20color&f=false) which features a case study of Tristan Harris, a co-founder of the Center for Humane Technology. This is definitely one of the most insidious hacks adopted by most social media platforms. I'm glad that you are including a feature to make this change.
